src/control/cmd/daos/acl.go

```diff
@@ -0,0 +1,324 @@
+//
+// (C) Copyright 2021 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package main
+
+/*
+#include <daos.h>
+#include <gurt/common.h>
+
+#include "daos_hdlr.h"
+#include "property.h"
+
+void
+free_strings(char **str, size_t str_count)
+{
+	int i;
+
+	for (i = 0; i < str_count; i++)
+		D_FREE(str[i]);
+}
```

Comment:
Should be removed?

Reply:
No, this is how cgo works. You can write standalone code in .c/.h files in the same directory as the go package, or you can write small helpers in a comment before the `import "C"` statement. This `free_strings()` helper is used down on line 50.

That having been said, I added this before I would up adding the cmd/daos/util.c file and I wouldn't be opposed to moving it there if it reduces confusion.